cmd/agent/main.go

```diff
@@ -61,10 +59,7 @@ func main() {
 			baseFactory := svc.MetricsFactory.
```

Comment:
```suggestion
			mFactory := svc.MetricsFactory.
```
this way you don't have to change sites where it's used